src/Compatibility/ApiDiff/Microsoft.DotNet.ApiDiff/Microsoft.DotNet.ApiDiff.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetToolMinimum)</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFramework>$(NetToolMinimum)</TargetFramework>
```

Reply:
If only we had some sort of analyzer to catch these.

Reply:
Fixed.